### PR TITLE
Blog post Prague 2021 schedule

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/prague/2021/news/announcing-schedule.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page, drop us an email at `prague@writethedocs.org <mailto:prague@writethedocs.org>`_.
+Or just show up on Sunday ready to introduce your project!
+
+Shirts still on sale
+--------------------
+
+The `Write the Docs Prague 2021 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.
+
+The campaign will run in 3-day batches until November 30th. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
```
Comment (mxsasha) [inline docs/conf/prague/2021/news/announcing-schedule.rst]: Is "sport your new threads" a widely known expression? I haven't heard it before. Clear enough from context though, so not a big deal.

**Review by mxsasha — APPROVED** [review]

File: docs/conf/prague/2021/news/announcing-schedule.rst (hunk 1 above)
Comment (thatdocslady) [reply docs/conf/prague/2021/news/announcing-schedule.rst]: Yeah I suppose it's a bit US-centric, but we've used it before... I don't mind changing it :)

Comment (plaindocs) [reply docs/conf/prague/2021/news/announcing-schedule.rst]: "show off your new gear/shirt" if we do want an alternative, but I agree that it isn't needed.

Comment (thatdocslady) [reply docs/conf/prague/2021/news/announcing-schedule.rst]:
```suggestion
The campaign will run in 3-day batches until November 30th. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to wear your new shirt during the conference.
```

File: docs/conf/prague/2021/news/announcing-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+All our events and activities
```
Comment (plaindocs) [inline docs/conf/prague/2021/news/announcing-schedule.rst]:
```suggestion
All conference events and activities
```

**Review by plaindocs — APPROVED** [review]
Nailed it!

You could switch out `prague` for `{{shortcode}}` if you wanted to make the URLs reusable, but it's all good, and probably hardly worth changing at this stage.

**Pull request merged**